src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -481,9 +486,14 @@ export default class PhotoField extends React.Component {
   }
 
   render() {
+    const { formData, formContext } = this.props;
+    const onReview = formContext.reviewMode;
+
+    if (onReview) return <PhotoReviewDescription formData={formData}/>;
```

Comment:
We might need to skip the resize event listener on review as well.

Reply:
Great idea, changed.